kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -125,29 +101,78 @@
     components: {
       'icon-button': require('kolibri.coreVue.components.iconButton'),
     },
-    props: [
-      'userid', 'username', 'fullname', 'roles', // TODO - validation
-    ],
+    props: {
+      userid: {
+        type: String, // string is type returned from server
```

Comment:
I believe all UIDs should be strings all the time. They might look like integers, but it's an illusion ;-) Basically, we should always treat them as opaque byte strings and never do any parsing or logic on them.